hidimstat/estimator/Dnn_learner_single.py

```diff
@@ -283,6 +284,9 @@ def encode_outcome(self, y, train=True):
             y = y.reshape(-1, 1)
         if self.problem_type == "regression":
             list_y.append(y)
+        # Encoding the target with the ordinal case
```

Comment:
@jpaillard @bthirion 
Can you tell me, if you know what is the "ordinal methods".
if yes, do you thing it's interesting to keep it? (the function was half implemented.) 
If it's interesting to  keep it, can you check if my modification was corrected?

Reply:
This will be address in the future. #76